starcheck/calc_ccd_temps.py

```diff
@@ -294,8 +305,8 @@ def get_week_states(tstart, tstop, bs_cmds, tlm):
     """
     Make states from last available telemetry through the end of the backstop commands
 
-    :param tstart: start time from first backstop command
-    :param tstop: stop time from last backstop command
+    :param tstart: start time from RLTT if available else first backstop command
+    :param tstop: stop time from SCHEDULED_STOP_TIME if available else last backstop command
```

Comment:
Give the description `schedule stop time from ...`

Reply:
The name should be `scheduled_stop`.  It doesn't need to be in seconds (tstop) at this point since it will be required as a date string later.

Reply:
Note that `tstop` is not actually even being used right now.  It needs to be used to set the `datestop` of the last state down below.  Something like:
```
states['datestop'][-1] = scheduled_stop
# Right after the get_states() call`
```